### PR TITLE
Changelog 0.5.0

### PR DESCRIPTION
**Pull request by jburel**
Changes for the coming release

cc @will-moore @pwalczysko @hflynn

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+1.0.0 (March 2018)
+------------------
+
+This version includes:
+
+
+- handled correctly data access depending on permissions level
```
Comment (hflynn) [inline CHANGELOG.md]: correctly handled

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+- added new icon for polyline
+- added minimal support for masks
+- improved UI
+- handled images with large number of channels
+- added support to open the image in other applications installed e.g. OMERO.figure
```
Comment (hflynn) [inline CHANGELOG.md]: to open the image in other applications installed -> for opening images in other installed applications

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+- added menus in header
+- improved rois creation and manipulation and persistence
```
Comment (hflynn) [inline CHANGELOG.md]: ROIs

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+- improved testing infrastructure
+- added option to export ROIs statictics in CSV file or Excel file
```
Comment (hflynn) [inline CHANGELOG.md]: statistics

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+- improved ROIs statistics export
+- updated documentation
+- improved loading and display of bird eye view
```
Comment (hflynn) [inline CHANGELOG.md]: birds eye

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,31 @@
...
+- displayed pixels intensity when mousing over image
+- enabled the viewing of multiple images in the same browser window by double-clicking
```
Comment (hflynn) [inline CHANGELOG.md]: enabled viewing 
(you don't need 'the')

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,31 @@
...
+- improved UI
+- handled images with large number of channels
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added menus in header
+- improved ROIs creation and manipulation and persistence
+- improved testing infrastructure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs statistics export
+- updated documentation
+- improved loading and display of birds eye view
```
Comment (hflynn) [inline CHANGELOG.md]: bird's 
(sorry that's my fault!)

But otherwise 👍 thanks

**Comment by hflynn** [comment]
👍 thanks

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,31 @@
...
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added menus in header
+- improved ROIs creation and manipulation and persistence
+- improved testing infrastructure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs statistics export
+- updated documentation
+- improved loading and display of bird's eye view
+- displayed pixels intensity when mousing over image
+- enabled viewing of multiple images in the same browser window by double-clicking
```
Comment (pwalczysko) [inline CHANGELOG.md]: put this item earlier in the list

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,31 @@
...
+- handled images with large number of channels
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added menus in header
+- improved ROIs creation and manipulation and persistence
+- improved testing infrastructure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs statistics export
+- updated documentation
+- improved loading and display of bird's eye view
+- displayed pixels intensity when mousing over image
```
Comment (pwalczysko) [inline CHANGELOG.md]: put this item earlier in the list

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,31 @@
...
+- correctly handled data access depending on permissions level
+- added new icon for polyline
+- added minimal support for masks
+- improved UI
+- handled images with large number of channels
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added menus in header
+- improved ROIs creation and manipulation and persistence
+- improved testing infrastructure
+- added option to export ROIs statistics in CSV file or Excel file
```
Comment (pwalczysko) [inline CHANGELOG.md]: put this item earlier in the list

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,31 @@
...
+
+
+- correctly handled data access depending on permissions level
+- added new icon for polyline
+- added minimal support for masks
+- improved UI
+- handled images with large number of channels
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added menus in header
+- improved ROIs creation and manipulation and persistence
```
Comment (pwalczysko) [inline CHANGELOG.md]: put this item earlier in the list

**Comment by jburel** [comment]
@pwalczysko better?

**Comment by pwalczysko** [comment]
>@pwalczysko better?

Ordering reads great now, thank you

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,31 @@
...
+This version includes:
+
+
+- displayed pixels intensity when mousing over image
+- enabled viewing of multiple images in the same browser window by double-clicking
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs creation and manipulation and persistence
+- added new icon for polyline
+- improved ROIs statistics export
```
Comment (will-moore) [inline CHANGELOG.md]: I'm not sure we need to have "Added ROIs statistics" and "Improved ROIs statistics" in the same list?

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,31 @@
...
+------------------
+
+This version includes:
+
+
+- displayed pixels intensity when mousing over image
+- enabled viewing of multiple images in the same browser window by double-clicking
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs creation and manipulation and persistence
```
Comment (will-moore) [inline CHANGELOG.md]: Replace first 'and' with a comma?

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,31 @@
+1.0.0 (March 2018)
+------------------
+
+This version includes:
+
+
+- displayed pixels intensity when mousing over image
```
Comment (will-moore) [inline CHANGELOG.md]: "display..." or "added option to display...."

File: CHANGELOG.md (hunk 12 above)
Comment (jburel) [reply CHANGELOG.md]: That one is more for the handling of the values.
It needs to be rephrased

**Comment by jburel** [comment]
@will-moore better?

**Comment by will-moore** [comment]
Looks good now, thanks

**Comment by hflynn** [comment]
I'll copy this content into the release announcement today now everyone is happy with it

**Comment by waxenegger** [comment]
@jburel not sure if these things need explicit, special mention (regarding most recent PRs): rois pasting and various bugfixes. if not I cannot think of anything else that was missed.

**Comment by jburel** [comment]
@hflynn: one more adjustment

**Comment by hflynn** [comment]
Website announcement PR updated now

**Pull request merged**